integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/management/RefreshCarbonTableCommand.scala

```diff
@@ -166,6 +178,11 @@ case class RefreshCarbonTableCommand(
       case e: Exception =>
         throw e
     }
```

Comment:
please do it in finally block, else in case of any exception, it will be set as true itself